SPEC.md

```diff
@@ -549,6 +551,16 @@ The AC Name Type is used as the primary key for a number of fields in the schema
 The schema validator will ensure that the keys conform to these constraints.
 
 
+### AC Version Type
+
+The App Container specification aims to follow semantic versioning and retain forward and backwards compatibility within major versions. For example, if an implementation is compliant against version 1.0.1 of the spec, it shall remain compatible with the complete 1.x series.
+
+The current version of the App Container spec is recorded in [VERSION](https://github.com/appc/spec/blob/master/VERSION), and is otherwise denoted in the [release version](https://github.com/appc/spec/releases) or git version control tag. 
+
+An AC Version (`acVersion`) must reference a tagged version of the App Container specification, not exceeding the version of its greatest compliance.
+An AC Version (`acVersion`) for [Image Manifest](#image-manifest-schema) and [Pod Manifest](#pod-manifest-schema) schemas must be on compatible version series.
```

Comment:
hmm, struggling with wording here. Maybe "to be used together"? something like that...